### PR TITLE
Simplified classList within Python code

### PR DESCRIPTION
**Pull request by WebReflection**
## Description

This MR would like to address this specific comment https://github.com/pyscript/pyscript/pull/1453#pullrequestreview-1414853366

## Changes

  * use a single JS <-> Python roundtrip to add or remove multiple classes

## Checklist

-   [x] All tests pass locally
-   [ ] I have updated `docs/changelog.md`
-   [ ] I have created documentation for this(if applicable)

### CODE REVIEW

**Comment by WebReflection** [comment]
on hold ... there might be some `.apply(...)` gotcha in here and, in such case, this MR shouldn't go through

File: pyscriptjs/src/python/pyscript/_html.py [hunk 1]
```diff
@@ -123,23 +123,23 @@ def clone(self, new_id=None, to=None):
...
         else:
-            self.element.classList.remove(classname)
+            classList.remove(classname)
 
     def add_class(self, classname):
+        classList = self.element.classList
         if isinstance(classname, list):
-            for cl in classname:
-                self.element.classList.add(cl)
+            classList.add.apply(classList, classname)
```
Comment (hoodmane) [inline pyscriptjs/src/python/pyscript/_html.py:135]:
```suggestion
            classList(*classname)
```

Comment (hoodmane) [inline pyscriptjs/src/python/pyscript/_html.py:135]:
```suggestion
            classList.add(*classname)
```

Comment (WebReflection) [reply pyscriptjs/src/python/pyscript/_html.py]: Done

**Review by madhur-tandon — APPROVED** [review]
LGTM

**Pull request merged**